### ISSUE TITLE
Audit accessibility

### ISSUE THREAD

**Issue by ahunt** [mozilla-mobile/focus-android#100]
We should make sure Focus is accessible.

I.e. we'll definitely want "contentDescription" set for ImageButtons, and we should do some testing with TalkBack.

**Comment by pocmo** [comment]
I started to do this for the new overflow menu and will continue to test the whole app.

**Comment by pocmo** [comment]
I added missing content descriptions and accessibility events.

I think the autocompleting URL bar could be better: You type 'm', 'o', 'z' and we autocomplete to 'mozilla.org' but Talkback doesn't announce that. So when pressing enter you actually go to mozilla.org instead of searching for 'moz' what you might had in mind.

**Comment by pocmo** [comment]
We should now have everything in place. Let's close this one and new ones for the specific issues we find.
